pyflakes/checker.py

```diff
@@ -1139,15 +1146,15 @@ def IMPORTFROM(self, node):
                 # Only Python 2, local import * is a SyntaxWarning
                 if not PY2 and not isinstance(self.scope, ModuleScope):
                     self.report(messages.ImportStarNotPermitted,
```

Comment:
This ImportStarNotPermitted now has a slightly unusual error message (as opposed to wrong before) as `from . import *` will generate 

    x.py:1: '..*' <error message>

`from .x import *` will generate the more readable

    x.py:1: '.x.*' <error message>

IMO '..*' is OK, but not ideal, so this presentation problem doesn't need to be fixed in this patch if we're rushing it out the door.

I will look into it more, but I am unsure how is should be displayed to be most readable.